tests/network/sriov.go

```diff
@@ -70,11 +70,7 @@ var _ = Describe("[Serial]SRIOV", func() {
 
 	var virtClient kubecli.KubevirtClient
 
-	sriovResourceName := os.Getenv("SRIOV_RESOURCE_NAME")
-
-	if sriovResourceName == "" {
-		sriovResourceName = "kubevirt.io/sriov_net"
-	}
+	sriovResourceName := readSRIOVResourceName()
```

Comment:
Can you please move this change to a commit of its own?
I find it different from this commit's subject.

Reply:
Done